news-bundle/contao/classes/News.php

```diff
@@ -30,85 +27,28 @@ class News extends Frontend
 	 *
 	 * @return string
 	 */
-	public static function generateNewsUrl($objItem, $blnAddArchive=false, $blnAbsolute=false)
+	public static function generateNewsUrl($objItem, $blnAddArchive=false, $blnAbsolute=true)
```

Comment:
Changing the default would be a BC break, so we must leave it as is.

Reply:
good catch! No idea why that's there, that was never intended. Reverted in 7fde1e1dcd73bf77e8827ae8b448754310aed3fa